ui/app/components/splash-page.hbs

```diff
@@ -0,0 +1,24 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{yield to="altContent"}}
+{{#unless @mfaErrors}}
```

Comment:
I think knowing about MFA errors is too specific for this component to know -- it should not care about MFA or any other specifics around how it's used. 

Instead you can check if there's an altContent block, and then otherwise show the rest of this. eg:
```
{{#if (has-block "altContent")}}
  {{yield to="altContent"}}
{{else}}
  ...default template
{{/if}}
```
but since we're not wrapping the altContent in any styling, I'm wondering what the value of having that here is. Not sure why we don't just show the alternate content in the parent